pkg/services/secrets/kvstore/helpers.go

```diff
@@ -27,3 +28,54 @@ func SetupTestService(t *testing.T) SecretsKVStore {
 
 	return kv
 }
+
+// In memory kv store used for testing
+type FakeSecretsKVStore struct {
```

Comment:
Not sure where my comment went to sorry if this is a duplicate... Would suggest we move this to the secrets/fakes directory for consistency.